Frameworks/CoreGraphics/CGFont.mm

```diff
@@ -110,7 +114,8 @@ CGFontRef CGFontCreateWithDataProvider(CGDataProviderRef cgDataProvider) {
     CFAutorelease(ret);
     struct __CGFont* mutableRet = const_cast<struct __CGFont*>(ret);
 
-    RETURN_NULL_IF_FAILED(_DWriteCreateFontFaceWithDataProvider(cgDataProvider, &mutableRet->_dwriteFontFace));
+    mutableRet->_data.reset(CGDataProviderCopyData(cgDataProvider));
+    RETURN_NULL_IF_FAILED(_DWriteCreateFontFaceWithData(mutableRet->_data.get(), &mutableRet->_dwriteFontFace));
```

Comment:
_have_ we been following that? on develop, the CoreText standard seems to be null return...

---
In reply to: [89845644](https://github.com/Microsoft/WinObjC/pull/1454#discussion_r89845644) [](ancestors = 89845644)